docs/source/cheat_sheet_py3.rst

```diff
@@ -125,16 +125,15 @@ See :ref:`async-and-await` for the full detail on typing coroutines and asynchro
            count -= 1
        return "Blastoff!"
 
-   # mypy currently does not support converting functions into coroutines in
-   # Python 3.4, so you need to add a 'yield' to make it typecheck.
+   # mypy currently does not support converting functions into async generators
```

Comment:
Async generators are a different feature that was added in 3.6. Do you mean generator-based coroutines?

Reply:
Yeah that is my bad. I suggested it and forgot about PEP 525. "generator based coroutine" is probably better.

Reply:
Fixed.